interoperator/Dockerfile

```diff
@@ -1,24 +1,34 @@
 # Build the manager binary
-FROM golang:1.11.2 as builder
+FROM golang:1.13.4 as builder
 
-# Copy in the go src
-WORKDIR /go/src/github.com/cloudfoundry-incubator/service-fabrik-broker/interoperator
-COPY . .
-# Install dep
-RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-RUN dep ensure
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY internal/ internal/
+COPY pkg/ pkg/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -tags provisioners -a -o manager github.com/cloudfoundry-incubator/service-fabrik-broker/interoperator/cmd/manager
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -tags schedulers -a -o scheduler github.com/cloudfoundry-incubator/service-fabrik-broker/interoperator/cmd/manager
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -tags multiclusterdeploy -a -o multiclusterdeploy github.com/cloudfoundry-incubator/service-fabrik-broker/interoperator/cmd/manager
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -tags provisioners -a -o manager main.go
```

Comment:
Why is this `GO111MODULE=on` required here? We do not have it in the GOPATH, right ?